generic-examples/http/README.md

```diff
@@ -38,6 +38,14 @@ Alternatively, you could run: `curl http://<service-location>/hello`.
 
 This integration requires a Keystore and a Truststore. Open [NettySecureServer.java](./NettySecureServer.java) to find instructions on how to generate a required `keystore.jks` and `truststore.jks` file. For this example, keystore and truststore password is `changeit`
 
+Generate keystore.jks and truststore.jks (for this example, keystore and truststore password = changeit):
+
+```shell
+keytool -genkeypair -alias EntryName -keyalg RSA -keysize 2048 -keystore keystore.jks
+keytool -exportcert -alias EntryName -keystore keystore.jks -rfc -file public.cert
+keytool -import -alias EntryName -file public.cert -storetype JKS -keystore truststore.jks
+```
+
```

Comment:
There is a TODO that we may remove in the following section. Aslo, we should better move the creation of the secret off the running Integration section below.

Reply:
ok, done